src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlParameter.cs

```diff
@@ -317,8 +330,8 @@ string xmlSchemaCollectionName
             SqlDbType = dbType;
             Size = size;
             Direction = direction;
-            PrecisionInternal = precision;
-            ScaleInternal = scale;
```

Comment:
The difference between precision and scale calculations between netcore and netfx was the main reason that I hadn't merged these two files. It looked to me as if the netcore version was later than the netfx version because netfx was simpler so netcore looked like a fix for something specific but it wasn't clear what behaviour it was fixing. If that's the case then this merge would regress some behaviour for someone.

It would be good to get some deep dive info on what the difference between the netfx and netcore behviour means for this.

Reply:
I have kept both framework properties to avoid any regression.